android/engine/src/main/java/org/smartregister/fhircore/engine/util/Constants.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.smartregister.fhircore.engine.util;
+
+public interface Constants {
+
+    String SLASH_UNDERSCORE = "/_";
```

Comment:
move this to same package where we have copied pasted server classes as is.. we may want to remove these classes from code soon and use as dependent library

Reply:
Moved